hnn_core/network_builder.py

```diff
@@ -306,11 +321,12 @@ def _build(self):
 
         self.state_init()
 
-        # set to record spikes and somatic voltages
+        # set to record spikes, somatic voltages, and extracellular potentials
         self._spike_times = h.Vector()
         self._spike_gids = h.Vector()
         self._vsoma = dict()
         self._isoma = dict()
+        self._rec_array = dict()
```

Comment:
I kept thinking elsewhere this was an instance of `ExtraCellularArray`. Perhaps `_extracellular_data` ?

Reply:
would `self._nrn_rec_array` work here too?